xmm-pysas-intro-short.ipynb

```diff
@@ -0,0 +1,212 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e10d847e-8f3d-4ad1-b0d4-c2427a7a7002",
+   "metadata": {},
+   "source": [
+    "# (Short) XMM-Newton pySAS Introduction\n",
+    "---\n",
+    "This tutorial provides a short, basic introduction to using pySAS on SciServer. A much more comprehensive introduction can be found in the <a href=\"./xmm-pysas-intro-long.ipynb\">Long pySAS Introduction</a> In this tutorial we will demonstrate,\n",
+    "\n",
+    "1. How to select a directory for data and analysis.\n",
+    "2. How to copy XMM data from the HEASARC archive.\n",
+    "3. How to run the standard XMM SAS commands `cfibuild` and `odfingest`.\n",
+    "\n",
+    "This tutorial uses an observation of NGC3079 (`obsid = '0802710101'`).\n",
+    "\n",
+    "The following assumes this notebook is run from the (xmmsas) environment on SciServer. You should see (`xmmsas`) at the top right of the notebook. If not, click there and select (`xmmsas`).\n",
+    "\n",
+    "##### *Last Reviewed: 6 November 2023, for SAS v21*\n",
+    "##### *Last Updated: 6 November 2023*\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "615f2005-6eb6-4c49-bae2-e44f15afba6b",
+   "metadata": {},
+   "source": [
+    "In the cell below you will need to enter your <code style=\"background:yellow;color:black\">user ID</code> so that your analyzed data will be placed in your `persistent` directory. You can set your data directory to any path you want, but if it is not inside the `persistent` directory your data will be lost upon exiting SciServer."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dbba53a7-a029-4aea-87a6-1e51a6423079",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "user = 'userid'\n",
+    "import gofpysas as pysas\n",
+    "data_dir = '/home/idies/workspace/Storage/{0}/persistent/xmm_data'.format(user)\n",
```

Comment:
I suggest using the current directory as `data_dir`. The user can change it if they want. I find asking for the user input here unnecessary. 
```python
data_dir = os.getcwd()
```